components/scream/src/share/io/scorpio_output.cpp

```diff
@@ -13,7 +13,7 @@ void AtmosphereOutput::init()
   // Parse the parameters that controls this output instance.
   // See the comments at the top for more details.
   m_casename        = m_params.get<std::string>("FILENAME");
-  m_grid_name       = m_params.get<std::string>("GRID","Physics");  // optional, default to Physics.
+  m_grid_name       = m_params.get<std::string>("GRID","Physics");  // optional, default to Physics., TODO: I don't think this is used anymore.
```

Comment:
Then let's remove it. If the code stops compiling or starts crashing, you can add it back. In general, let's aim at keeping classes lean. If something is suspected to be unused, let's remove it. It's relatively easy to add small things like this back, if need be.